### PR TITLE
Add devise gem for authentication

### PR DESCRIPTION
**Pull request by yuliatokaryk**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :models
```
Comment (ihortok) [inline config/routes.rb:2]: why model?

File: app/models/model.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+class Model < ApplicationRecord
```
Comment (ihortok) [inline app/models/model.rb:1]: why model?

**Review by ihortok — CHANGES_REQUESTED** [review]

File: app/models/model.rb (hunk 2 above)
Comment (yuliatokaryk) [reply app/models/model.rb]: 117e651fcf89600c86a39ac33fe777b44474cae1

File: config/routes.rb (hunk 1 above)
Comment (yuliatokaryk) [reply config/routes.rb]: 117e651fcf89600c86a39ac33fe777b44474cae1

**Review by ihortok — APPROVED** [review]

**Pull request merged**